gemelli/q2/tests/test_transformer.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import qiime2
+import numpy as np
+import pandas.util.testing as pdt
+from skbio.util import get_data_path
+from qiime2.plugin.testing import TestPluginBase
+from gemelli.q2._format import TrajectoryFormat
+
+
+class TestTrajectoryFormatTransformers(TestPluginBase):
+    package = "gemelli.q2.tests"
+
+    def test_pd_series_to_first_differences_format(self):
+
+        transformer = self.get_transformer(pd.DataFrame, TrajectoryFormat)
+        test_df = pd.DataFrame(np.random.normal(size=(10, 3)),
+                               [str(i) for i in range(10)],
+                               ['PC1', 'PC2', 'PC3'])
+        test_df.index.name = '#SampleID'
+        result = transformer(test_df)
+        if isinstance(result, str):
+            pass
```

Comment:
This test does not check for anything

Reply:
it just checks that the transformed runs.

Reply:
Okay, then you should remove
```
        if isinstance(result, str):
            pass
```
since it does nothing

Reply:
Maybe you meant to do:
```
self.assertIsInstance(result, str)
```
?